skin/frontend/base/default/js/TIG/MyParcel2014/myparcel.js

```diff
@@ -157,8 +158,10 @@ MyParcel = {
         }
 
         /* Titels of the options*/
-        if (MyParcel.data.config.deliveryTitel){
+        mypajQuery('#mypa-delivery-title').hide();
+        if (MyParcel.data.config.allowMorningDelivery || MyParcel.data.config.allowStandardDelivery || MyParcel.data.config.allowEveningDelivery){
             mypajQuery('#mypa-delivery-titel').html(MyParcel.data.config.deliveryTitel);
+            mypajQuery('#mypa-delivery-title').show();
```

Comment:
mag op 1 regel

Reply:
Mag niet op 1 regel `mypa-delivery-titel` en `mypa-delivery-title` was er gebruikt (dit is nu aangepast).